packages/coding-standards/src/CsFixer/Phpdoc/NoUselessAccessFixer.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\CodingStandards\CsFixer\Phpdoc;
+
+use PhpCsFixer\DocBlock\DocBlock;
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\Fixer\FixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+final class NoUselessAccessFixer implements FixerInterface, DefinedFixerInterface
+{
+    private const ACCESS_TOKENS = [
+        \T_PUBLIC,
+        \T_PROTECTED,
+        \T_PRIVATE,
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            '`@access` annotations should be omitted from PHPDoc when it is useless',
+            [
+                new CodeSample(
+                    '<?php
+class Foo
+{
+    /**
+     * @internal
+     * @access private
+     */
+    private $bar;
+}
+'
+                ),
+            ]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens): bool
+    {
+        return $tokens->isTokenKindFound(\T_DOC_COMMENT) && $tokens->isAnyTokenKindsFound(self::ACCESS_TOKENS);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens): void
+    {
+        foreach ($tokens->findGivenKind(\T_DOC_COMMENT) as $index => $token) {
```

Comment:
This function is quite nested, could you extract inside of the foreach into a separated function?

Reply:
Hope it's better now :)